src/blog/posts/views.py

```diff
@@ -71,9 +65,9 @@ def delete_post(post_id: PrimaryKey):
             detail=[{"msg": "Post with this id doesn't exist"}],
         )
     delete(post_id=post_id)
-        # get post by id using service method
-        # if not post raise 404 HTTPException (check specification: https://fastapi.tiangolo.com/tutorial/handling-errors/ )
-        # else post exists, call delete from service
-        # usually no need to return anything from delete, sometimes the deleted id can be returned,
-        # but it's ok to return just HTTP 200 response
-        # (If there are no exceptions raised and no return, FastAPI automatically return HTTP 200)
+    # get post by id using service method
```

Comment:
I think now you can remove all this comments.

Reply:
All comments are removed.